backend/src/test/java/com/ody/member/controller/MemberControllerTest.java

```diff
@@ -0,0 +1,52 @@
+package com.ody.member.controller;
+
+import com.ody.auth.token.AccessToken;
+import com.ody.common.BaseControllerTest;
+import com.ody.common.FixtureGenerator;
+import com.ody.common.TokenFixture;
+import com.ody.mate.domain.Mate;
+import com.ody.meeting.domain.Meeting;
+import com.ody.member.domain.Member;
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Import;
+import org.springframework.http.HttpHeaders;
+
+@Import({FixtureGenerator.class})
```

Comment:
[제안]
하나인 경우 `{}`생략가능합니다!
```suggestion
@Import(FixtureGenerator.class)
```

Reply:
[1af7ae1](https://github.com/woowacourse-teams/2024-ody/pull/512/commits/1af7ae1acafb63f24a3eee0017b02984fd77fd84)